lib-object-oriented/lexer/identifiers_md/_factory.js

```diff
@@ -0,0 +1,15 @@
+import Asterisk from './asterisk.js'
+import Bang from './bang.js'
+import Hash from './hash.js'
+import Underscore from './underscore.js'
+
+export default class Factory {
+
+
+  switch(symbol) {
```

Comment:
Is switch a method of factory? I am confused about what this switch is doing here.